### DISCUSSION TITLE
Duda sobre elemento de stadisticas

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#414] [Q&A] [answered]
Buena tarde compañero me gustaría saber como pinto en mi html el jso.

![poke dex json](https://user-images.githubusercontent.com/68626555/158470474-0e1b7800-1f2d-4d61-afac-0f806af67c90.png)

y mi código lo tengo a si el js cree un id en mi html para que me lo tomara pero no me lo pinto

![js](https://user-images.githubusercontent.com/68626555/158470665-b8bdedaf-9c7f-4c86-95ad-f8ddb5804eba.png)

![codi](https://user-images.githubusercontent.com/68626555/158470911-4f998934-75d1-4ec1-9024-6be55b50d9e3.png)

**Comment by AngelCruzO** [comment, marked as answer]
En vez de src, pon innerHTML

**Reply by gersondlarosa** [reply]
Lo voy a hacer

**Reply by jricardocam** [reply]
Hola, me despliega con el inner HTML, pero me toma el ultimo valor para todos

**Reply by AngelCruzO** [reply]
Yo lo que hice fue antes de que me retorne el json, volver a asignarle un valor vacío, para resetear el valor. Stats.innerHTML = ""

**Comment by ChristianDeM** [comment]
para cambiar el html lo hice con un .textContent por ejemplo

` <h2 id="attack"></h2>`
` document.getElementById("attack").textContent= (`${attack}`);`

lleva comillas dobles el  ${attack}  con alt +96 salen jeje
bueno en mi caso me funciono a si

**Reply by gersondlarosa** [reply]
Lo voy a hacer

**Reply by erickmatz9101** [reply]
Disculpa te puedo hacer una pregunta sobre tú codigo? osea que ya solo mandaste a llamar lo elementos donde estan es decir cuando llamas al pokemon te sale la imagen porque en los sprites esta ese elemento y veo que de igual manera mandaste a llamar a lo demas solo teniendo en cuenta cual era la ubicacion?

**Reply by gersondlarosa** [reply]
Si como ya esta en la url los mando a llamar con el ley y lo que quiero traer del json no se si sea buena practica pero a si me los trae

**Reply by erickmatz9101** [reply]
gracias por contestar
